resource-watch/resource-watch
# next@8.0.4

## Overview
Includes major update of `next`. Updates the applicaton from `next 7` to `next 8.1.0`. just setting foundation for future `next` features.

More info: https://nextjs.org/blog/next-8

## Testing instructions
Type `rm -rf node_modules && yarn` to run a clean dependency installation.
`yarn dev` to check everything is fine.

## Pivotal task
–

---

## Checklist before submitting
